engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformula/ringbuffervectorwrapper/CharRingBufferVectorWrapper.java

```diff
@@ -0,0 +1,48 @@
+package io.deephaven.engine.table.impl.updateby.rollingformula.ringbuffervectorwrapper;
+
+import io.deephaven.base.ringbuffer.CharRingBuffer;
+import io.deephaven.vector.CharVector;
+import io.deephaven.vector.CharVectorDirect;
+
+public class CharRingBufferVectorWrapper implements CharVector, RingBufferVectorWrapper {
+    private final CharRingBuffer ringBuffer;
+
+    public CharRingBufferVectorWrapper(final CharRingBuffer ringBuffer) {
+        this.ringBuffer = ringBuffer;
+    }
+
+    @Override
+    public long size() {
+        return ringBuffer.size();
+    }
+
+    @Override
+    public char get(long index) {
+        return ringBuffer.front((int)index);
+    }
+
+    @Override
+    public CharVector subVector(long fromIndexInclusive, long toIndexExclusive) {
+        throw new UnsupportedOperationException("subVector not supported on CharRingBufferVectorWrapper");
+    }
+
+    @Override
+    public CharVector subVectorByPositions(long[] positions) {
+        throw new UnsupportedOperationException("subVectorByPositions not supported on CharRingBufferVectorWrapper");
```

Comment:
Can use the existing slice and subvector classes to implement this. Not insanely efficient, but works...